packages/composite-checkout/src/components/wp-checkout-order-review.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import joinClasses from '../lib/join-classes';
+import { useLineItems, renderDisplayValueMarkdown } from '../public-api';
+import {
+	OrderReviewLineItems,
+	OrderReviewTotal,
+	OrderReviewSection,
+} from './order-review-line-items';
+import Coupon from './coupon';
+
+export default function WPCheckoutOrderReview( { summary, className } ) {
+	const [ items, total ] = useLineItems();
+
+	if ( summary ) {
```

Comment:
I'm pretty sure you got rid of this component having a summary version here: https://github.com/Automattic/wp-calypso/pull/37261/files#diff-5b63d2418405e222d04228a455428b98L252

Reply:
Good call!